lib/CMakeLists.txt

```diff
@@ -44,124 +39,173 @@ add_library(arango_static_strings STATIC
   Basics/StaticStrings.cpp
 )
 
+set(FROZEN_INCLUDE_DIR "${PROJECT_SOURCE_DIR}/3rdParty/iresearch/external/frozen/include")
+set(FUNCTION2_INCLUDE_DIR "${PROJECT_SOURCE_DIR}/3rdParty/iresearch/external/function2")
+
```

Comment:
Maybe an explanation what arangdob_lightweight is for wouldn't go amiss?

This seems to be the place in the code where that could live.